frontend/apps/app/src/functions/postComment.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  createPullRequestComment,
+  updatePullRequestComment,
+} from '@/libs/github/api'
+import { prisma } from '@liam-hq/db'
+import type { ReviewResponse } from '../types'
+
+export async function postComment(
+  payload: ReviewResponse,
+): Promise<{ success: boolean; message: string }> {
+  try {
+    const { reviewComment, pullRequestId, repositoryId } = payload
+
+    // Get repository information
+    const repository = await prisma.repository.findUnique({
+      where: {
+        id: repositoryId,
+      },
+    })
+
+    if (!repository) {
+      throw new Error(`Repository with ID ${repositoryId} not found`)
+    }
+
+    // Check if there's an existing PR record with a comment
+    const prRecord = await prisma.pullRequest.findUnique({
+      where: {
+        repositoryId_pullNumber: {
+          repositoryId,
+          pullNumber: BigInt(pullRequestId),
+        },
+      },
+    })
```

Comment:
I don't think pullNumber == pullRequestId, so should we look for it from pullRequestId?👀
```
    const prRecord = await prisma.pullRequest.findUnique({
      where: {
        id: payload.pullRequestId,
      },
    })
```

Reply:
You're right 🙏🏻 I'll fix it!

Reply:
fixed it https://github.com/liam-hq/liam/pull/886/commits/553d05b36f5368fe0ad7a6d1d03641a1ce9b618a
Thanks!!

Reply:
Ah, there may be a few more places that need to be corrected.🙏

https://github.com/liam-hq/liam/blob/68ad466444a0d39a43298a6b3d165e88c7a88d10/frontend/apps/app/src/functions/postComment.ts#L57-L68

Reply:
Oh... sorry, I'll fix too 🙏🏻

Reply:
Overall, mistakes were made and have been corrected!
https://github.com/liam-hq/liam/pull/886/commits/bd17ae203b93a120f0b187c92d01d4ca8bcc6682

Reply:
Thank you!!👍